requirements.txt

```diff
@@ -1,16 +1,12 @@
 configobj>=5.0.5
-cherrypy==8.9.1
+cherrypy>=3.6.0,<9.0
 ws4py>=0.3.2
-SQLAlchemy>=1.0.0
+SQLAlchemy>=1.1.0
 six>=1.5.2
 Jinja2>=2.7
 rpctools>=0.3.1
 logging_unterpolation>=0.2.0
 requests>=2.2.1
-pytest>=3.0.1
-mock>=1.0.1,<1.1
-Sphinx>=1.2.1
-coverage>=3.6
```

Comment:
For reference, we are doing this exact thing in ubersystem and all of the other plugins. The tests for each plugin ensure that necessary packages are installed, and do not rely on sideboard installing testing packages, so the plugin tests should be fine.

Also, I used the conventions `requirements_*.txt` everywhere else. So we've got:
* `requirements.txt`
* `requirements_docs.txt`
* `requirements_tests.txt`

And those files reference each other so we don't have to list the same core requirements three times. I think the way we have it set up is pretty nice, so feel free to use that as inspiration.